charts/castai-cluster-controller/templates/deployment.yaml

```diff
@@ -71,6 +74,9 @@ spec:
         - name: cluster-controller
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
command is not required for this image?

Reply:
For backward compatibility, I made a controller option by default (no command specified) to cover cases when, on the latest helm version, you downscale the image version